Squeezer/src/main/java/uk/org/ngo/squeezer/service/SqueezeService.java

```diff
@@ -654,23 +654,20 @@ private void startForeground() {
 
             ongoingNotification = notificationState();
             NotificationData notificationData = new NotificationData(ongoingNotification);
-            Notification notification;
+            Notification notification = notificationData.builder.build();
 
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                final MediaMetadataCompat.Builder metaBuilder = new MediaMetadataCompat.Builder();
-                metaBuilder.putString(MediaMetadata.METADATA_KEY_ARTIST, ongoingNotification.artistName);
-                metaBuilder.putString(MediaMetadata.METADATA_KEY_ALBUM, ongoingNotification.albumName);
-                metaBuilder.putString(MediaMetadata.METADATA_KEY_TITLE, ongoingNotification.songName);
-                mMediaSession.setMetadata(metaBuilder.build());
-                notification = notificationData.builder.build();
-            } else {
-                notification = notificationData.builder.build();
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
Android studio said this was always true so I removed the condition.

Reply:
Yes let's git rid of that, I just haven't gotten around to this one yet.